### PR TITLE
Fix #7899: Remove SwiftKeychainWrapper dependency.

### LINKED ISSUE #7899: Remove SwiftKeychainWrapper dependency
We do not use it anymore

### PR DESCRIPTION
**Pull request by iccub**
## Summary of Changes

This pull request fixes #7899 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`
- [x] New or updated UI has been tested across:
  - [x] Light & dark mode
  - [x] Different size classes (iPhone, landscape, iPad)
  - [x] Different dynamic type sizes

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Comment by nuo-xu** [comment]
<strike>gotta remove from project file as well. Client -> Package Dependencies</strike>
Resolved. thank you

**Comment by iccub** [comment]
thank you Nuo

File: Sources/Brave/Assets/About/Licenses.html [hunk 1]
```diff
@@ -686,39 +686,6 @@ <h4 id="exhibit-b---incompatible-with-secondary-licenses-notice">Exhibit B - “
 
 <p class="center">-</p>
 
-<div>
```
Comment (soner-yuksel) [inline Sources/Brave/Assets/About/Licenses.html:689]: Can you also update line 846 where Feedkit is written ZIP foundation

```
    <input id="ac-FeedKit" name="accordion-1" type="checkbox" />
    <label for="ac-FeedKit"><h2>ZIP Foundation</h2></label>
```

    <input id="ac-FeedKit" name="accordion-1" type="checkbox" />
    <label for="ac-FeedKit"><h2>FeedKit</h2></label>

**Review by soner-yuksel — APPROVED** [review]

File: Sources/Brave/Assets/About/Licenses.html (hunk 1 above)
Comment (iccub) [reply Sources/Brave/Assets/About/Licenses.html:689]: Made the change thanks. And removed `pop` library from licenses

**Pull request merged**